### PR TITLE
Addon 29030 update filter to collect data in f5 syslog

### PR DESCRIPTION
**Pull request by dhruvinshah-crest**

### CODE REVIEW

File: package/etc/conf.d/log_paths/lp-f5_bigip.conf.tmpl [hunk 1]
```diff
@@ -14,7 +14,13 @@ filter f_f5_bigip {
     or program("tmm\d?")
     or program('^f5_irule=')
     or program('F5')
-    or message('^f5_asm=Splunk-F5-ASM');
+    or message('^f5_asm=Splunk-F5-ASM')
```
Comment (rfaircloth-splunk) [inline package/etc/conf.d/log_paths/lp-f5_bigip.conf.tmpl:17]: the following lines don't reliably indicate the event is F5 as the terms could be used by other products. We must relay on the host match to set vendor_product then classify inside the log path

**Review by rfaircloth-splunk — CHANGES_REQUESTED** [review]
see comments

File: package/etc/conf.d/log_paths/lp-f5_bigip.conf.tmpl (hunk 1 above)
Comment (nkaleiya) [reply package/etc/conf.d/log_paths/lp-f5_bigip.conf.tmpl:17]: @rfaircloth-splunk, As per our Splunk doc about the irules configuration, it is mentioned that f5_asm should be equal to Splunk-F5-ASM. Reference: https://docs.splunk.com/Documentation/AddOns/released/F5BIGIP/Setup

Comment (rfaircloth-splunk) [reply package/etc/conf.d/log_paths/lp-f5_bigip.conf.tmpl:17]: Line 17 wasn't changed the comment applies to the new lines below

Comment (dhruvinshah-crest) [reply package/etc/conf.d/log_paths/lp-f5_bigip.conf.tmpl]: Yes true, Understand your point and removed all those matches as we already have a match for the host on the fist line as `match("^f5_bigip", value("fields.sc4s_vendor_product"))`.

**Review by rfaircloth-splunk — APPROVED** [review]

**Pull request merged**